src/graph/label_graph.py

```diff
@@ -4,13 +4,12 @@
 
 from src.utils.common import chunkify
 
-MAX_MATRIX_SIZE = 1000000
-
 
 class LabelGraph:
```

Comment:
It used to be better and clearer. Firstly, it is necessary to preserve the possibility of a constructor of zero arguments (`xs = LabelGraph()`). Secondly - the number of edges should definitely not appear in the constructor. It is a parameter, that **can be changed**. If you want to get the number of edges, you have to create a function for this (may be `get_number_of_edges` will return the sum of nnz of matrices). So I suggest you go back to the way it was and make `get_number_of_edges` to return sum of nnz.